### PR TITLE
Update picker to autocomplete

### PR DESCRIPTION
**Pull request by danmitchell-**
This repo is backed by the accessible autocomplete so therefore picker
isn’t the write terminology.

@J-Lambo can I get some input please

### CODE REVIEW

**Comment by J-Lambo** [comment]
Looks sensible to me!

File: README.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
 
 The `location-picker-graph.json` file only contains examples of synonyms, abbreviations, endonyms and typos you might want to consider. It is not a comprehensive list. You may wish to add or remove items based on your own user research.
 
 Copy both files to your application. The `location-picker-graph.json` file must be exposed as a public asset.
 
-You can also install the location picker using `npm`:
+You can also install the location autocomplete using `npm`:
 
 ```bash
-$ npm install openregister-location-picker
```
Comment (nickcolley) [inline README.md]: Do we need to republish this to npm? Is there a concept of a redirect in npm?

**Comment by danmitchell-** [comment]
Is that a seal of approval @J-Lambo ? 🤓

File: README.md (hunk 1 above)
Comment (danmitchell-) [reply README.md]: I think so @nickcolley, yes. Can you help? 

https://www.npmjs.com/package/openregister-location-picker

File: README.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
-# Location picker - what it is and how to use it
+# GOV.UK Country and Territory Autocomplete - what it is and how to use it
```
Comment (edwardhorsford) [inline README.md]: This should be sentence case

File: README.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
 
-[![Build Status](https://travis-ci.org/alphagov/openregister-location-picker.svg?branch=master)](https://travis-ci.org/alphagov/openregister-location-picker)
-[![Greenkeeper badge](https://badges.greenkeeper.io/alphagov/openregister-location-picker.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/alphagov/govuk-country-and-territory-autocomplete.svg?branch=master)](https://travis-ci.org/alphagov/govuk-country-and-territory-autocomplete)
+[![Greenkeeper badge](https://badges.greenkeeper.io/alphagov/govuk-country-and-territory-autocomplete.svg)](https://greenkeeper.io/)
 
-The location picker is a simple UI widget that allows end users to find and select a location. It includes auto-complete functionality to make it faster and easier for users to find a location. For example, to select their country of birth or their current location.
+The GOV.UK Country and Territory Autocomplete is a simple UI widget that allows end users to find and select a location. It includes auto-complete functionality to make it faster and easier for users to find a location. For example, to select their country of birth or their current location.
```
Comment (edwardhorsford) [inline README.md]: Should be sentence case

File: README.md (hunk 1 above)
Comment (nickcolley) [reply README.md]: The patterns and tools team are publishing GOV.UK stuff to npm, be good to check in with them 👍

Comment (nickcolley) [reply README.md]: Think we'll need to deprecate the current one and then publish the other as per pattern and tools advice.

https://docs.npmjs.com/cli/deprecate

Comment (danmitchell-) [reply README.md]: Not quiet as elegant as Github but yes, I agree

Comment (danmitchell-) [reply README.md]: Going to revert this change until we publish a new package

**Comment by danmitchell-** [comment]
If there are no more comments, can I get an approval @edwardhorsford @J-Lambo @nickcolley ?

**Comment by edwardhorsford** [comment]
I've not tested the changes, but I approve of the changes in principle.

I didn't know it was the GOV.UK Registers (not GDS team) - should this be in a separate PR anyway?

File: README.md (hunk 1 above)
Comment (danmitchell-) [reply README.md]: Actually we are going to publish a new package to npm now 👍

Comment (danmitchell-) [reply README.md]: Sorted @nickcolley 

```
$ npm install openregister-location-picker
npm WARN deprecated openregister-location-picker@0.3.2: Move to govuk-country-and-territory-autocomplete
```

Comment (nickcolley) [reply README.md]: Nice work @danmitchell- 🙌

**Pull request merged**